### PR TITLE
Convert hand rolled XML document generation to use the ElementTree module.

### PR DESCRIPTION
**Pull request by kevinsteves**
Functionally equivalent to previous with exceptions:

    remove XML declaration as not needed

    change config version to 8.1.0 (earliest supported PAN-OS)

    don't indent XML as this can be performed with other tooling
    (e.g., xmllint)

    application groups are not currently used and the XML was
    not rendered correctly, so remove XML generation for now.

### CODE REVIEW

File: capirca/lib/paloaltofw.py [hunk 1]
```diff
@@ -726,209 +726,166 @@ def __str__(self):
               etree.SubElement(icmp_type_props, "type").text = app[k][prop]
             else:
               pass
-
-    lines = minidom.parseString(
-        etree.tostring(app_entries)).toprettyxml(indent="  ")
```
Comment (rdsharma) [inline capirca/lib/paloaltofw.py:731]: Can we add this functionality back in?

**Review by rdsharma — CHANGES_REQUESTED** [review]

File: capirca/lib/paloaltofw.py (hunk 1 above)
Comment (kevinsteves) [reply capirca/lib/paloaltofw.py:731]: Added ident of the XML document.

**Review by rdsharma — APPROVED** [review]
This looks good, nice work! Will wait until @abhindes reviews as well (for syntax correctness) to pull this in.

Like I mentioned before, we'll have to do something to expose the etree object to tests later on so that we can check the DOM itself rather than the rendered output, but that can come in a later CL if you prefer.

**Review by abhindes — APPROVED** [review]
Looks good to me. I loved how easy it was to view the diffs as well.

Great work Kevin!

**Pull request merged**